Definitions.md

```diff
@@ -0,0 +1,16 @@
+#Definition Of Terms
+
+##Job
```

Comment:
Is my understanding here that Job is unlike the other three in that unlike Task, Workflow, or Tool, a Job is actually the only term here that is "actually run." In that there could be many Jobs which are effectively invocations of Tasks with different inputs and parameters?

If so, another name for Job (perhaps an alias?) might be TaskInstance which could imply symmetry with WorkflowInstance for example.